lms/envs/common.py

```diff
@@ -1248,9 +1248,9 @@
     'js/vendor/jquery-ui.min.js',
     'js/vendor/jquery.qtip.min.js',
     'js/vendor/jquery.ba-bbq.min.js',
-    'js/vendor/afontgarde/modernizr.fontface-generatedcontent.js',
-    'js/vendor/afontgarde/afontgarde.js',
-    'js/vendor/afontgarde/edx-icons.js'
+    # 'pattern-library/js/modernizr.custom.js',
```

Comment:
I agree that we shouldn't bundle these. Let's only load them via RequireJS, either through an optimized RequireJS factory bundle, or dynamically if necessary (such as with the video player). Go ahead and remove these commented out lines.